docs/source/adventure/project.rst

```diff
@@ -104,6 +106,391 @@ stick with a simple text-based intro. Try to recreate the following:
                 self.play(Unwrite(intro), Unwrite(vec_txts), run_time=0.5)
                 self.wait(0.2)
 
+============================
+Algebraic vector addition
+============================
+
+Then, let's show the viewer how vector addition between two vectors is done algebraically.
+Once again, try to recreate the following:
+
+
+.. manim:: AlgebraicAddition
+    :hide_source:
+    :ref_classes: Title MathTex Paragraph Tex Text Write Unwrite Create FadeIn
+    
+    class AlgebraicAddition(Scene):
+
+        def construct(self):
+            title = Title("Vector Addition Algebraically")
+
+            v1x, v1y = (2, 2)
+            v2x, v2y = (0, -3)
+            math = MathTex(r"""
+                \begin{bmatrix} %(v1x)d \\ %(v1y)d \end{bmatrix}
+                +\begin{bmatrix} %(v2x)d \\ %(v2y)d \end{bmatrix}
+            """ % {
+                'v1x': v1x,
+                'v2x': v2x,
+                'v1y': v1y,
+                'v2y': v2y
+            }).shift(DOWN)
+
+            resultant_vector = r"=\begin{bmatrix} %(x)d \\ %(y)d \end{bmatrix}" % {
+                'x': v1x+v2x,
+                'y': v1y+v2y
+            }
+            math_with_answer = MathTex(
+                math.get_tex_string()+resultant_vector
+            ).move_to(math.get_center())
+
+            self.play(Write(math), FadeIn(title))
+            self.wait(2)
+            self.play(
+                math.animate.shift(2*UP).set_opacity(0.5),
+                Write(math_with_answer)
+            )
+            conclusion = Paragraph("As you can see,\nYou add each component individually").to_edge(DOWN)
+            self.play(Write(conclusion))
+            self.wait(2)
+            self.play(Unwrite(math), Unwrite(math_with_answer), Unwrite(conclusion), Unwrite(title))
+
+Hints
+-----
+
+Use :class:`.Title` to display the title at the top of the Scene.
+
+Use :class:`.MathTex` to represent the matrices, try not to hardcode the values into the LaTeX string, instead, use python string formatting, this will allow us to define the resultant vector dynamically.
```

Comment:
nitpick: it's kind of hard to understand because of the run-on sentence.
What about something like this?
```suggestion
Use :class:`.MathTex` to represent the matrices, and try not to hardcode the values into the LaTeX string. Instead, you can use python string formatting and numpy vector addition, which will make it easier to change the vectors later if we need to.
```